package.json

```diff
@@ -50,13 +51,16 @@
   "devDependencies": {
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "chai": "^4.1.2",
     "chai-http": "^4.0.0",
     "coveralls": "^3.0.1",
     "eslint": "^5.4.0",
     "eslint-config-airbnb": "^17.1.0",
```

Comment:
`eslint-config-airbnb` and `eslint-plugin-jsx-a11y` not needed if we are using eslint-config-airbnb-base

Reply:
okay removing them now.

Reply:
I have done this.

Reply:
I have done this.